### ISSUE TITLE
Would you consider a PR to switch to tool installers

### ISSUE THREAD

**Issue by sandorfr** [Schlumberger/vsts-chromium-task#4]
Hi,

Would you consider accepting a PR switching from powershell to node engine. That would allow the extension to leverage tool installer APIs.

https://github.com/Microsoft/vsts-tasks/blob/master/docs/tools.md

Kind regards,
Cyprien

**Comment by sandorfr** [comment]
Also please consider merging #3 and removing the [Preview] flag from the title. (the marketplace already shows a preview label because of the flags).

**Comment by helgejfo** [comment]
I would accept a PR changing to node. The preview flag was added to validate the current script for a while before removing it. I would like to keep the preview for the new solution as well, for a while at least to validate the changes.

**Comment by sandorfr** [comment]
Yes my comment was not about the preview flag but the [preview] in the extension title. I think it's a bit redundant.
